packages/openjdk-17/hooks/debian/postinst

```diff
@@ -0,0 +1,21 @@
+#!@TERMUX_PREFIX@/bin/sh
+
+binaries="$(find @TERMUX_PREFIX@/lib/jvm/java-17-openjdk/bin -executable -type f | xargs -I{} basename "{}" | xargs echo)"
+manpages="$(find @TERMUX_PREFIX@/lib/jvm/java-17-openjdk/man/man1 -name "*.1.gz" | xargs -I{} basename "{}" | xargs echo)"
```

Comment:
This will now be executed on device instead of during build. This means that any junk that user has in $PREFIX/lib/jvm/java-17-openjdk will also be found and added to update-alternatives. This is especially a problem for prerm script, maybe list of binaries has changed when user has used openjdk, and prerm script will then fail because it tries to `update-alternatives --remove` something that has not been added as an alternative

It would be better if list of binaries and manpages are generated in build.sh and then sed'ed into postinst and prerm script